docs/User guide/README.md

```diff
@@ -0,0 +1,7 @@
+# User guide
```

Comment:
What options do we have for not having paths with capitals and spaces in them? as the folder structure then varies from the resto of the repo.

Reply:
Have updated the naming to kebab case to match the rest of the repo.

The labels in the sidebar nav for pages are pulled from the H1 given at the start of the page (if present), and for directories it seems to be converted to sentence case.

I kept the `API` dir as uppercase, as otherwise it was converted to `Api` in the sidebar.

It also has the benefit of using the file names as the URL, so for example `empty-tiles.md` becomes `http://0.0.0.0:8000/operator-guide/empty-tiles/` with a label of `Sparse Datasets and empty tiles` in the sidebar